src/modules/StorageProposal.rb

```diff
@@ -6095,7 +6101,14 @@ def get_proposal_vm(target, key, disk)
         )
         Builtins.y2milestone("get_proposal_vm sol:%1", disk)
       end
-      Builtins.y2milestone("get_proposal_vm ret:%1", ret)
+
+      post_processor = PostProcessor.new()
+      ret["target"] = post_processor.process_target(ret["target"])
+
+      Builtins.y2milestone(
+        "get_proposal_vm ret[ok]: %1",
+        Ops.get_boolean(ret, "ok", false)
```

Comment:
This is copied verbatim from the other places where this is used. Since there is no reasonable way to test this isolated, I tried to be minimal-invasive with those changes.